build/scala-client/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN set -ex && \
     apt-get clean && \
     rm -rf scala-${SCALA_VERSION}.deb sbt-${SBT_VERSION}.tgz /var/lib/apt/lists/*
 
+ENV PATH="/sbt/bin:$PATH"
```

Comment:
The Dockerfile installs sbt to this location, so it should deal with setting PATH, not the build script.